doc/lua_api.txt

```diff
@@ -140,14 +152,13 @@ Mod directory structure
     |   |-- sounds
     |   |-- media
     |   `-- <custom data>
-    `-- another
-
+    `-- another`
```

Comment:
This list style should be changed a bit. I think this looks a bit strange in a graphical markdown view.